ledger/ledger-api-test-tool-on-canton/BUILD.bazel

```diff
@@ -81,12 +81,7 @@ conformance_test(
             "ParticipantPruningIT",  # see "conformance-test-participant-pruning" below
             "ClosedWorldIT",  # Canton currently fails this test with a different error (missing namespace in "unallocated" party id)
             "CommandServiceIT:CSsubmitAndWaitCompletionOffset",  # Canton does not fill the offsets
```

Comment:
I can't find this test being excluded in Canton's own tests. So maybe this exclusion is now also obsolete?

Reply:
Similarly for `PartyManagementServiceIT:PMAllocateEmptyExpectMissingDisplayName` below.